repositories/user.js

```diff
@@ -1,17 +1,25 @@
-const User = require("../db/models/User")
-
+const User = require("../db/models/User");
+const jwt = require('jsonwebtoken');
 
 async function getUserByEmail(email) {
-    return User.findOne({email: email});
+  return User.findOne({email: email});
 }
 
 async function createUser(userData) {
-    const { email, password, role } = { ...userData }
+  const { email, password, role } = { ...userData }
+
+  return await User.create({email: email, password: password, role: role})
+}
 
-    return await User.create({email: email, password: password, role: role})
+async function setResetPasswordToken(user) {
```

Comment:
Why you doing this in repository. Repository only need to communicate with db. Move it in some service file